x/dymns/client/cli/tx_accept_offer.go

```diff
@@ -0,0 +1,89 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/dymensionxyz/dymension/v3/app/params"
+	dymnstypes "github.com/dymensionxyz/dymension/v3/x/dymns/types"
+	"github.com/spf13/cobra"
+)
+
+// NewAcceptDymNameBuyOfferTxCmd is the CLI command for accepting a Buy-Order of a Dym-Name
+// or offer-back to raise the price.
+func NewAcceptDymNameBuyOfferTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("accept-offer [offer-id] [amount] %s", params.DisplayDenom),
+		Short: "Accept a Buy-Order for your Dym-Name",
+		Example: fmt.Sprintf(
+			"$ %s tx %s accept-offer 1 50 %s --%s hub-user",
+			version.AppName, dymnstypes.ModuleName,
+			params.DisplayDenom,
+			flags.FlagFrom,
+		),
+		Args: cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			offerId := args[0]
+			if !dymnstypes.IsValidBuyOfferId(offerId) {
+				return fmt.Errorf("input Offer-ID '%s' is not a valid Offer-ID", offerId)
+			}
+
+			amount, err := strconv.ParseUint(args[1], 10, 64)
```

Comment:
this CLI usage looks like this: dymd tx dymns accept-offer 1 50 DYM`
So that's why I use Parse Int, because I need to parse the value 50, it's not 50adym, it's 50 DYM.